feg/gateway/docker/python/Dockerfile

```diff
@@ -82,7 +82,7 @@ RUN apt-get -y update && apt-get -y install \
   libjansson4 \
   libjemalloc2 \
   libssl-dev \
-  libsystemd-dev \
+  libsystemd-dev=245.4-4ubuntu3.18 \
```

Comment:
magma-preferences is not used in this Dockerfile, i.e., should not have a need to be pinned